tasks.robot

```diff
@@ -15,18 +29,16 @@ Split orders file
     ${table}=    Read worksheet as table    header=True
     ${groups}=    Group table by column    ${table}    Name
     FOR    ${products}    IN    @{groups}
-        Create output work item
+        Create Output Work Item
         ${rows}=    Export table    ${products}
         Set work item variable    products    ${rows}
-        Save work item
+        Save Work Item
```

Comment:
Do you think we might want to call after this the following: `Release input work item    DONE`.

We get an auto-loaded initial input work item, we create for it output work items, but then nobody sets its state. Not sure if setting (or forgetting to set) a state here has noticeable side-effects, but wanted to highlight it.